util/package_watcher.go

```diff
@@ -0,0 +1,83 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package util
+
+import (
+	"log"
+	"os"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/radovskyb/watcher"
+)
+
+const watcherPollingPeriod = time.Second
+
+var (
+	w *watcher.Watcher
+
+	watchedPackages []Package
```

Comment:
Why not use the same package list cache from packages.go? It seems the other one is not used if this is enabled.

Reply:
Fixed.